utilities/cli/holohub.py

```diff
@@ -857,7 +857,11 @@ def handle_test(self, args: argparse.Namespace) -> None:
             else:
                 image_name = args.base_img or container.default_base_image()
             tag = image_name.split(":")[-1]
-        ctest_cmd = f"{xvfb} ctest -DAPP={args.project} -DTAG={tag} "
+
+        ctest_cmd = f"{xvfb} ctest "
+        if args.project:
+            ctest_cmd += f"-DAPP={args.project} "
```

Comment:
Is this to allow users to request components explicitly with configuration args (`-DOP`, etc)? Or could you please comment on motivation?